superset/jinja_context.py

```diff
@@ -97,6 +97,8 @@ def process_template(self, sql, **kwargs):
         >>> process_template(sql)
         "SELECT '2017-01-01T00:00:00'"
         """
+        # Escaping colon
+        sql = sql.replace(':', '\:')
```

Comment:
Seems like the escaping is only required for some database engines as well. The specs being loose and the sqlalchemy dialects being of varying quality it makes this really hard on our end...